tests/NetEscapades.EnumGenerators.Tests/SourceGenerationHelperSnapshotTests.cs

```diff
@@ -20,8 +20,8 @@ public Task GeneratesEnumCorrectly()
             isPublic: true,
             new List<(string Key, EnumValueOption Value)>
             {
-                ("First", new EnumValueOption(null, false)),
-                ("Second", new EnumValueOption(null, false)),
+                ("First", new EnumValueOption(null, false, 0)),
+                ("Second", new EnumValueOption(null, false, 1)),
```

Comment:
This should give modified snapshots, right?

Reply:
No, because they are in ascending order. The underlying value is only used in ordering, the output is `ShortName.First` or `ShortName.Second` whenever we want the respective (underlying) value